README.md

```diff
@@ -1,172 +1,54 @@
-If using Traceur, these can be set with:
-
-```javascript
-System.traceurOptions = {...};
-```
-
-Or with Babel:
-
-```javascript
-System.babelOptions = {...};
-```
-
-#### Module Tag
-
-As well as defining `window.System`, this polyfill provides support for the `<script type="module">` tag:
-
-```html
-<script type="module">
-  // loads the 'q' export from 'mymodule.js' in the same path as the page
-  import { q } from 'mymodule';
-
-  new q(); // -> 'this is an es6 class!'
-</script>
-```
-
-Because it is only possible to load ES6 modules with this tag, it is not suitable for production use in this way.
-
-See the [demo folder](https://github.com/ModuleLoader/es6-module-loader/blob/master/demo/index.html) in this repo for a working example demonstrating module loading in the browser both with `System.import` and with the module-type script tag.
-
-#### NodeJS Use
-
-```
-  npm install es6-module-loader babel traceur
-```
-
-It is important that Babel or Traceur is installed into the path in order to be found, since these are no longer project dependencies.
-
-For use in NodeJS, the `Loader` and `System` globals are provided as exports:
-
-index.js:
-```javascript
-  var System = require('es6-module-loader').System;
-  /*  
-   *  Include:
-   *    System.transpiler = 'babel'; 
-   *  to use Babel instead of Traceur
-   */
-
-  System.import('some-module').then(function(m) {
-    console.log(m.p);
-  });
-```
-
-some-module.js:
-```javascript
-  export var p = 'NodeJS test';
-```
-
-Running the application:
-```
-> node index.js
-NodeJS test
-```
+Dynamically loads ES6 modules in browsers and NodeJS with support for loading existing and custom module formats through loader hooks.
+
+### Specifications
+ES6/ES2015 introduces native support for Javascript modules, self-standing pieces of code which execute in their own scope and import values exported by other modules. Originally, the specification included the loader, and this polyfill was created to support those loader functions in current browsers and in NodeJS.
+
+The final specification, though, only defines the [language-specific semantics of modules](http://www.ecma-international.org/ecma-262/6.0/#sec-modules), leaving implementations of the loading of those modules to separate specifications. The dynamic configurable loader is being specified at the [WhatWG loader spec](https://whatwg.github.io/loader/), and from version 0.50 this polyfill implements that.
+
+Support for static loading of modules from HTML using `<script type="module">` tags has also been added to the [WhatWG HTML specification](https://html.spec.whatwg.org/multipage/scripting.html#the-script-element). This adapts the existing script-loading mechanism to include the loading of 'module scripts' together with all the other modules on which those module scripts depend. Unlike the WhatWG loader, however, this is not configurable, and has no connection with the loading pipeline defined there and implemented in this polyfill.
```

Comment:
Where have you seen the term `module scripts` used? _Module tag_ is preferable I think.